.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
   matrix:
     # Run scikit-learn tests as integration / non-regression tests
     - SKIP_TESTS="true" SKLEARN_TESTS="true" PYTHON_VERSION="3.6" NUMPY_VERSION="1.14"
-    - PYTHON_VERSION="2.7" NUMPY_VERSION="1.6"
+    - PYTHON_VERSION="2.7" NUMPY_VERSION="1.6" NO_LZ4="true" COVERAGE="true"
```

Comment:
Why did you add coverage?

Reply:
To cover [this line](https://github.com/joblib/joblib/pull/740/files#diff-41146308443e078486d3c42ea7798174R1067)

Reply:
ok